app/src/client/hooks/useBuildPageNew.ts

```diff
@@ -1,21 +1,20 @@
 import { useState, useEffect } from 'react';
+import { PropertiesSchema } from '../interfaces/BuildPageInterfacesNew';
+import { getSchema } from 'wasp/client/operations';
 
-import { getAvailableModels } from '../services/modelService';
-
-import { ApiResponse } from '../interfaces/BuildPageInterfaces';
-
-export const useBuildPage = () => {
-  const [data, setData] = useState<ApiResponse | null>(null);
+export const useBuildPageNew = () => {
```

Comment:
in the future, since old `useBuildPage` no longer exists, maybe we can revert back  and call  `useBuildPageNew` just `useBuildPage`

Reply:
I have updated the filename from `useBuildPageNew` to `useBuildPage`.